lib/Flux.js

```diff
@@ -126,18 +215,36 @@ class Flux {
     return first;
   }
 
+  /**
+   * Fetch the content of a store given his path.
+   * @param {*} binding TODO: To implement after signature change.
```

Comment:
We decided to use explicit parameters instead of `...binding`. Documentation will be updated according to new paramaters.